xarray/tests/test_backends.py

```diff
@@ -3923,6 +3923,12 @@ def test_ENVI_tags(self):
                 assert isinstance(rioda.attrs["map_info"], str)
                 assert isinstance(rioda.attrs["samples"], str)
 
+    def test_geotiff_tags(self):
+        # Create a geotiff file with some tags
+        with create_tmp_geotiff() as (tmp_file, _):
+            with xr.open_rasterio(tmp_file) as rioda:
+                assert isinstance(rioda.attrs["AREA_OR_POINT"], str)
```

Comment:
Is `"AREA_OR_POINT"` set by default? I don't see it in `create_tmp_geotiff()`

Reply:
Should we check for a specific value instead of just the type?

Reply:
Yes, `AREA_OR_POINT` is set by default in a geotiff http://web.archive.org/web/20160326194152/http://remotesensing.org/geotiff/spec/geotiff2.5.html#2.5.2

Reply:
> Should we check for a specific value instead of just the type?

I don't think that is needed, whatever is there needs to be reported for interpretation. XArray doesn't act on that value.